orttraining/orttraining/core/graph/mixed_precision_transformer.cc

```diff
@@ -70,10 +66,30 @@ bool IsFP32(const std::unordered_map<std::string, std::vector<int>>& map, std::s
   }
 }
 
+static const std::string Loss_Scale_Input = "loss_scale";
```

Comment:
>Loss_Scale_Input [](start = 25, length = 16)

Nit: loss_scale_input?

(and same for loss_subgraph_entry_nodes and loss_subgraph_exit_nodes?)